.github/workflows/prepare-release.yml

```diff
@@ -16,8 +16,8 @@ jobs:
           git-message: "chore(release): {version}"
           tag-prefix: ""
           output-file: "CHANGELOG.md"
-          version-file: "pyproject.toml"
-          version-path: "tool.poetry.version"
+          version-file: "package.json"
+          version-path: "version"
```

Comment:
doesn't seem right, is a python template

Reply:
I thought I fixed it... Fixed now!